### PR TITLE
self.training removed

### PR DESCRIPTION
**Pull request by pranoyr**
I guess the self.training  in the forward is not allowing to create the zeros mask for the null logits during inference.

### CODE REVIEW

**Comment by lucidrains** [comment]
@pranoyr oops, thanks!
